tokio-util/Cargo.toml

```diff
@@ -43,6 +44,9 @@ futures-util = { version = "0.3.0", optional = true }
 pin-project-lite = "0.2.11"
 slab = { version = "0.4.4", optional = true } # Backs `DelayQueue`
 tracing = { version = "0.1.25", default-features = false, features = ["std"], optional = true }
+nix = "0.21"
+rand = "0.8"
```

Comment:
We don't add new dependencies to Tokio without a very very good reason. Please use `libc` instead of `nix`. Furthermore, Tokio already comes with its own random number generator (for `select!`), so you can use that instead of `rand`.